src/Model/Todo/Handler/NotifyUserOfExpiredTodoHandler.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * This file is part of prooph/proophessor-do.
+ * (c) 2014-2016 prooph software GmbH <contact@prooph.de>
+ * (c) 2015-2016 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Prooph\ProophessorDo\Model\Todo\Handler;
+
+use Prooph\ProophessorDo\Model\Todo\Command\NotifyUserOfExpiredTodo;
+use Prooph\ProophessorDo\Model\Todo\Query\GetTodoById;
+use Prooph\ProophessorDo\Model\User\Query\GetUserById;
+use Prooph\ServiceBus\QueryBus;
+use Zend\Mail\Message;
+use Zend\Mail\Transport\TransportInterface;
+
+class NotifyUserOfExpiredTodoHandler
+{
+    /**
+     * @var QueryBus
+     */
+    private $queryBus;
+
+    /**
+     * @var TransportInterface
+     */
+    private $mailer;
+
+    /**
+     * @param QueryBus $queryBus
+     * @param TransportInterface $mailer
+     */
+    public function __construct(
+        QueryBus $queryBus,
+        TransportInterface $mailer
+    ) {
+        $this->queryBus = $queryBus;
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * @param NotifyUserOfExpiredTodo $command
+     *
+     * @return void
+     */
+    public function __invoke(NotifyUserOfExpiredTodo $command)
+    {
+        $todo = null;
+        $this->queryBus->dispatch(new GetTodoById($command->todoId()->toString()))
+            ->then(
+                function ($result) use (&$todo) {
+                    $todo = $result;
+                }
+            );
+        $user = null;
+        $this->queryBus->dispatch(new GetUserById($todo->assigne_id))
+            ->then(
+                function ($result) use (&$user) {
+                    $user = $result;
+                }
+            );
+
+        $message = sprintf(
+            'Hi %s! Just a heads up: your todo `%s` has expired on %s.',
+            $user->name,
+            $todo->text,
+            $todo->deadline
+        );
+
+        $mail = new Message();
+        $mail->setBody($message);
+        $mail->setEncoding('utf-8');
+        $mail->setFrom('reminder@getprooph.org', 'Proophessor-do');
```

Comment:
The "FROM" address should be configurable.
Anyway, while this is only an "example" application, i am fine if it is not configurable.

BUT:
The default address should not be "getprooph.org". This application runs on client pc´s and could accidentally be used to spam others. So "getprooph" should not be the Spamer.